modules/services/chunkwm.nix

```diff
@@ -0,0 +1,142 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.chunkwm;
+
+in
+
+{
+  options = {
+    services.chunkwm.enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = "Whether to enable the chunkwm window manager.";
+    };
+
+    services.chunkwm.package = mkOption {
+      type = types.package;
+      default = pkgs.chunkwm-core;
+      defaultText = "pkgs.chunkwm-core";
+      description = "This option specifies the chunkwm package to use";
+    };
+
+    services.chunkwm.hotload = mkOption {
+      type = types.bool;
+      default = true;
+      description = "Whether to enable hotload";
+    };
+
+    services.chunkwm.port = mkOption {
+      type = types.int;
+      default = 3920;
+      description = "ChunkC port";
+    };
+
+    services.chunkwm.plugins.dir = mkOption {
+      type = types.path;
+      default = "/run/current-system/sw/lib/chunkwm/plugins";
```

Comment:
To clarify, I was talking about `/run/current-system/sw/share/chunkwm/plugins`. But it's fine if that doesn't work